### ISSUE TITLE
Fatal error will be reported when sending for the second time

### ISSUE THREAD

**Issue by xyxywan** [koen-github/WakeOnLan-ESP8266#9]
use ESP32,The first execution completed normally

 //(Second execution of the command WakeOnLan::sendWOL(computer_ip, UDP, mac, sizeof mac);)
21:38:27.139 -> exec cmd..wol       
21:38:27.139 -> CORRUPT HEAP: Bad head at 0x3ffd6e48. Expected 0xabba1234 got 0x3ffd7768
21:38:27.139 -> abort() was called at PC 0x40086a09 on core 1
21:38:27.139 -> 
21:38:27.139 -> ELF file SHA256: 0000000000000000
21:38:27.139 -> 
21:38:27.139 -> Backtrace: 0x400885c0:0x3ffb1900 0x4008883d:0x3ffb1920 0x40086a09:0x3ffb1940 0x40086b35:0x3ffb1970 0x400f7ac7:0x3ffb1990 0x400f3db5:0x3ffb1c50 0x400f3d44:0x3ffb1ca0 0x4008cf19:0x3ffb1cd0 0x40081c56:0x3ffb1cf0 0x40086901:0x3ffb1d10 0x4000bec7:0x3ffb1d30 0x4015d105:0x3ffb1d50 0x4015ce45:0x3ffb1d70 0x400d4177:0x3ffb1d90 0x400d4243:0x3ffb1dc0 0x400d42f1:0x3ffb1e00 0x400d14e1:0x3ffb1e30 0x400d29a2:0x3ffb1ea0 0x400d8c40:0x3ffb1fb0 0x4008984e:0x3ffb1fd0
21:38:27.186 -> 
21:38:27.186 -> Rebooting...

**Comment by koenieee** [comment]
Still an issue?

**Comment by xyxywan** [comment]
Yes, when this function is executed twice in a row, the device must go wrong and restart
